src/server/command_registry.h

```diff
@@ -73,6 +73,16 @@ static_assert(!IsEvalKind(""));
 // Per thread vector of command stats. Each entry is {cmd_calls, cmd_latency_agg in usec}.
 using CmdCallStats = std::pair<uint64_t, uint64_t>;
 
+struct CommandContext {
```

Comment:
Just a thought: since this will be used quite a lot, maybe rename it to `CmdContext` or `CmdCntx`?
Feel free to say no obviously, I just don't think it makes the code less readable.